client-src/elements/chromedash-roadmap.ts

```diff
@@ -314,7 +314,7 @@ export class ChromedashRoadmap extends LitElement {
         milestone => html`
           <chromedash-roadmap-milestone-card
             style="width:${this.cardWidth}px;"
-            .channel=${this.milestoneInfo[milestone]}
+            .channel=${this.milestoneInfo ? this.milestoneInfo[milestone] : null}
```

Comment:
I think this could be shorter as:
`.channel=${this.milestoneInfo?.[milestone]}`